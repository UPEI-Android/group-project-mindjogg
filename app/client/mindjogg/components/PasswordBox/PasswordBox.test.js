import React from "react";

describe("Textbox", () => {
    it("should render", () => {
        expect(true).toBeTrue();
    });
});
