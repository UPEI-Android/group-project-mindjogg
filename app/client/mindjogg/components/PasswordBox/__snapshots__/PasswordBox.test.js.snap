// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textbox should render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "borderColor": "#000",
      "borderRadius": 5,
      "borderWidth": 0.5,
      "flexDirection": "row",
      "height": 40,
      "justifyContent": "center",
      "margin": 10,
    }
  }
>
  <Text />
  <input
    maxLength={1000}
    minLength={1}
    name="myPasswordBox"
    onChange={[Function]}
    required={true}
    style={
      Object {
        "borderStyle": "none",
        "display": "block",
        "outline": "none",
      }
    }
    type="password"
    value=""
  />
  <input
    maxLength={1000}
    minLength={1}
    name="myPasswordBox"
    onChange={[Function]}
    required={true}
    style={
      Object {
        "borderStyle": "none",
        "display": "none",
        "outline": "none",
      }
    }
    type="text"
    value=""
  />
  <Text />
</View>
`;
